src/Post_Type_Command.php

```diff
@@ -34,6 +34,37 @@ class Post_Type_Command extends WP_CLI_Command {
 		'capability_type',
 	);
 
+	/**
+	 * Gets the post counts for each supplied post type.
+	 *
+	 * @param array $post_types Post types to fetch counts for.
+	 * @return array Associative array of post counts keyed by post type.
+	 */
+	protected function get_counts( $post_types ) {
+		global $wpdb;
+
+		if ( count( $post_types ) <= 0 ) {
+			return array();
```

Comment:
We're at PHP 5.4+ now, so we should use short array syntax.
```suggestion
			return [];
```